provisioner/env.go

```diff
@@ -62,6 +65,11 @@ var (
 func getOpenEBSNamespace() string {
 	return menv.Get(menv.OpenEBSNamespace)
 }
+
+func getNfsServerNamespace() string {
+	return menv.Get(NFSServerNamespace)
```

Comment:
This probably should handle the case where NFSServerNamespace is not present in the YAML and return the same value configured to getOpenEBSNamespace if empty.